doc/man3/EC_GROUP_copy.pod

```diff
@@ -181,9 +181,17 @@ EC_GROUP_method_of returns the EC_METHOD implementation in use for the given cur
 
 EC_GROUP_get0_generator returns the generator for the given curve or NULL on error.
 
-EC_GROUP_get_order, EC_GROUP_get_cofactor, EC_GROUP_get_curve_name, EC_GROUP_get_asn1_flag, EC_GROUP_get_point_conversion_form
-and EC_GROUP_get_degree return the order, cofactor, curve name (NID), ASN1 flag, point_conversion_form and degree for the
-specified curve respectively. If there is no curve name associated with a curve then EC_GROUP_get_curve_name will return 0.
+EC_GROUP_get_order() return the order as outparams from the given GROUP.
```

Comment:
In this case for example a good `RETURN VALUES` entry would be:

```pod
EC_GROUP_get_order() returns 0 if the order is not set (or set to zero) for
B<group> or if copying into B<order> fails, 1 otherwise.
```

And the corresponding `DESCRIPTION` entry could be similar to the following:

```pod
EC_GROUP_get_order() retrieves the order of B<group> and copies its value into
B<order>.  It fails in case B<group> is not fully initialized (i.e., its order
is not set or set to zero).
```

Reply:
@vilvarajintel would you be willing to update also the `DESCRIPTION` entries that are missing?

Reply:
I will update the Descriptions as well